.github/ISSUE_TEMPLATE/support.yml

```diff
@@ -0,0 +1,36 @@
+name: Question/Help/Support
+description: Ask us about Manim Slides
+title: "Support: Ask us anything"
+labels: ['help', 'question']
+
+body:
+  - type: dropdown
+    attributes:
+      label: Criticality
+      description: >
+        Please select the criticality of the support you're requesting for
+
+        1 - highest priority
+        5 - Lowest priority
+        <details>
+          <summary>
+            <em>Priority?</em>
+          </summary>
+        </details>
+      options:
+        - "1"
+        - "2"
+        - "3"
+        - "4"
+        - "5"
+    validations:
+      required: true
+
```

Comment:
```suggestion
```
I think it's the reviewer's job to define how critical an issue is, not the user's.